src/pages/components/tile/usage.mdx

```diff
@@ -259,38 +293,372 @@ contain separate internal CTAs.
 
 ### Selectable
 
-Selectable tiles work like a [radio button](/components/radio-button/usage),
-where the entire tile is a click target. Selectable tiles may contain internal
-CTAs (like links to docs) if the internal CTA is given its own click target.
 Selectable tiles work well for presenting options to a user in a structured
-manner, such as a set of pricing plans.
+manner, such as a set of pricing plans. Selectable tiles may contain internal
+CTAs (like links to docs) if the internal CTA is given its own click target.
+
+Selectable tiles can either have a single select state working like a
+[radio button](/components/radio-button/usage), or multi-select state working as
+a [checkbox](/components/checkbox/usage).
 
 <Row>
 <Column colLg={8}>
 
-![Example image of selectable tiles.](images/tile-usage-3.png)
+![Selectable radio-button / single select tiles](images/tile-usage-3.png)
+
+<Caption>Selectable radio-button / single select tiles.</Caption>
+
+![Selectable checkbox / multi-select tiles](images/tile-usage-4.png)
+
+<Caption>Selectable checkbox / multi-select tiles. </Caption>
 
 </Column>
 </Row>
 
 ### Expandable
 
-Expandable tiles are helpful for hiding/showing larger amounts of content to a
-user. They can only be stacked in a single column, and cannot live in a row or
-horizontal grid. When expanded, tiles push content down the page. Expandable
-tiles may contain internal CTAs (like links to docs) if the internal CTA is
-given its own click target.
+Expandable tiles are helpful for hiding and showing large amounts of content to
+a user.  
+When expanded, tiles push content down the page. They allow the user to
+specifically focus on featured content while having access to the rest of the
+information. Expandable tiles can contain internal CTAs (like links to docs) if
+they are given their own click targets and the click target is reduced to only
+the chevron icon.
+
+<Row>
+<Column colLg={8}>
+
+![Example image of expandable tiles.](images/tile-usage-5.png)
+
+</Column>
+</Row>
+
+## Formatting
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Example image of expandable tiles.](images/tile-usage-6.png)
+
+</Column>
+</Row>
+
+1. **Header**: Includes a title, optional label, and the close icon.
+
+2. **Body**: Contains the information and/or controls needed to complete the
+   modal's task. It can include message text and components.
+
+### Sizing
+
+The **width** varies depending on three basic grid modes: wide, narrow, and
+condensed while the **height** varies depending on the amount of content placed
```

Comment:
is it because it's a run-on sentence? Should it be Meanwhile, instead of While?

Reply:
It’s because you use the colon in the first part of the sentence. I think you’ve got to end the sentence after listing the items after the colon. You could switch the colon to a comma. I’m not entirely sure from a grammar point on this one but it seemed weird to have a colon in the middle of a sentence and then have the sentence continue.

Reply:
Asked Jan for her opinion on this. She said it’s a bit long when the two concepts are linked together in the one sentence. Just a lot to hold in your head while you’re reading. so she suggested this:

**"The width varies depending on three basic grid modes: wide, narrow, and condensed. The height varies depending on the amount of content placed within it, but still follows the aspect ratio."**